guides/common/modules/ref_best-practices-for-sync-plans.adoc

```diff
@@ -0,0 +1,24 @@
+[id="best-practices-for-sync-plans_{context}"]
+= Best practices for sync plans
+
+* Add sync plans to products and regularly synchronize content to avoid overly large changes in repositories.
```

Comment:
Why are "overly large changes" bad?

Reply:
It causes e.g. bigger load on Foreman Server during synchronization. Smaller diffs in remote repos is easier to handle.

Reworded.